### ISSUE TITLE
Low Accessibility due to missing iframe

### ISSUE THREAD

**Issue by sayedhajaj** [weknowinc/gatsby-remark-codepen#3]
According to Google Lighthouse, there is a noticeable decline in accessibility after adding a codepen embed. One of the suggestions was to add a title.

**Comment by jmolivas** [comment]
Fixed with https://github.com/weknowinc/gatsby-remark-codepen/pull/4
